Documentation/BuildInstructions.md

```diff
@@ -31,10 +31,12 @@ Now on Ubuntu or Debian you can install gcc-11 with apt like this:
 sudo apt install gcc-11 g++-11
 ```
 
-#### QEMU 5 or later
+#### QEMU 6.2.0 or later
 
-QEMU version 5 is available in Ubuntu 20.10, but it is recommended to build Qemu as provided by the toolchain by running `Toolchain/BuildQemu.sh`.
-Note that you might need additional dev packages:
+Version 6.0 of Qemu is available as a package in Ubuntu 21.10, but version 6.2.0 or later is required to run
```

Comment:
I'd just leave out the part about Ubuntu 21.10, I don't think there is a point in specifically mentioning that an outdated Ubuntu version provides an outdated QEMU verison. If you want to mention which QEMU version is provided by Ubuntu, do so for the newest LTS (i.e. 22.04), which provides QEMU 6.2.

The remaining text can stay as-is in my opinion, except for maybe replacing `Qemu` with `QEMU`.

Reply:
Thanks for your review -- I have updated the PR. I force-pushed the branch, which I realize may have been a faux pas. (I saw that the commit-lint CI step had failed, probably due to a longer-than-72-chars commit message, and I wasn't sure whether the PR would be merge-able with that lint failure -- new contribution jitters)

Reply:
Force-pushes are absolutely fine. :^)